ansible/deploy-all.yaml

```diff
@@ -1,35 +1,38 @@
 ---
 ############################################
+# Third-party
+############################################
+- include: infra-amqp-brokers.yaml
+  when: not parasitic
+
+- include: infra-deploy-irods-elasticsearch.yaml
+  when: not parasitic
+
+############################################
```

Comment:
In that case, I don't think we should be including private playbooks from public playbooks.